app/models/mentor/application.rb

```diff
@@ -112,7 +162,10 @@ def sql_statement_all
           teams.name AS team_name,
           projects.name AS project_name,
           (application_data -> :project_id)::int AS project_id,
-          CASE WHEN :project_id::text = 'project1_id' THEN TRUE ELSE FALSE END AS first_choice
+          application_data -> :signed_off_at AS signed_off_at,
```

Comment:
we might consider moving the existing indexes from the old columns to the new columns. I don't think it's urgent but we should not forget about it, since it can has an impact on the application performance.

old indexes
```
application_data -> project1_id
application_data -> project2_id
signed_off_by
```
possible new indexes, just theoretical, we should look at the query in production (ideally development staging :D)
```
application_data -> :signed_off_at
application_data -> :signed_off_by
application_data -> :mentor_fav
```